doc/contributors/guides/how_to_make_driver.md

```diff
@@ -0,0 +1,476 @@
+# How to Make a Puter Driver
```

Comment:
I understand why you moved this. Can we keep it where it is for now? There are links to this document in many other places so moving this to the correct location will have to happen later.